.github/workflows/continous-integration.yml

```diff
@@ -191,7 +191,7 @@ jobs:
     needs: [changes]
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest]
```

Comment:
can we add windows back now?

Reply:
yes, I think this is @tczekajlo addition?

Reply:
done